benchmarks/dynamo/ci_expected_accuracy/update_expected.py

```diff
@@ -0,0 +1,140 @@
+"""
+Update commited CSV files used as reference points by dynamo/inductor CI.
+
+Currently only cares about graph breaks, so only saves those columns.
+
+Hardcodes a list of job names and artifacts per job, but builds the lookup
+by querying github sha and finding associated github actions workflow ID and CI jobs,
+downloading artifact zips, extracting CSVs and filtering them.
+
+Usage:
+
+1) get a rocksdp API key and save it in ~/rocks_api_key
+2) python benchmarks/dynamo/ci_expected_accuracy.py <sha of pytorch commit that has completed inductor benchmark jobs>
+
+Known limitations:
+- doesn't handle 'retry' jobs in CI, if the same hash has more than one set of artifacts, gets the first one
+"""
+
+import argparse
+import os
+import urllib
+from io import BytesIO
+from urllib.request import urlopen
+from zipfile import ZipFile
+
+import pandas as pd
+import requests
+
+
```

Comment:
nit: I'd suggest adding a link to the artifacts query in rockset or something. The goal is to let a future dev find the exact query that this is pointing to, since it's really hard to do that reverse search with this public url